sensu-plugins-pagerduty.gemspec

```diff
@@ -23,12 +23,12 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.3.0'
```

Comment:
We need to call this out as a breaking change see https://github.com/sensu-plugins/sensu-plugins-redis/pull/69/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10 for an example.

Reply:
updated the same